operator-framework/src/main/java/io/javaoperatorsdk/operator/ControllerUtils.java

```diff
@@ -48,34 +52,11 @@ static String getCrdName(ResourceController controller) {
     public static <T extends CustomResource> Class<? extends CustomResourceDoneable<T>>
     getCustomResourceDoneableClass(ResourceController<T> controller) {
         try {
-            Class<? extends CustomResource> customResourceClass = getAnnotation(controller).customResourceClass();
-            String className = customResourceClass.getPackage().getName() + "." + customResourceClass.getSimpleName() + "CustomResourceDoneable";
-
-            if (doneableClassCache.containsKey(customResourceClass)) {
```

Comment:
@csviri @kirek007 @adam-sandor 
Do you think this caching is still needed?

Reply:
Doneable classes are going away.